src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -707,20 +705,22 @@ export class DotEditContentHtmlService {
     }
 
     private renderRelocatedContentlet(relocateInfo: any): void {
-        const doc = this.getEditPageDocument();
-        const contenletEl = doc.querySelector(
-            `div[data-dot-object="contentlet"][data-dot-inode="${relocateInfo.contentlet.inode}"]`
-        );
-        const contentletContentEl = contenletEl.querySelector('.dotedit-contentlet__content');
-        contentletContentEl.innerHTML +=
-            '<div class="loader__overlay"><div class="loader"></div></div>';
-        relocateInfo.container =
-            relocateInfo.container || contenletEl.parentNode.dataset.dotIdentifier;
-
-        this.dotContainerContentletService
-            .getContentletToContainer(relocateInfo.container, relocateInfo.contentlet)
-            .subscribe((contentletHtml: string) => {
-                this.appendNewContentlets(contenletEl, contentletHtml);
-            });
-    }
+        if (!relocateInfo.remoteRendered) {
```

Comment:
we should do this here:

https://github.com/dotCMS/core-web/blob/19d5f21823c236d4c7192848a4bdc6669888d989/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts#L610